swupd/swupd.go

```diff
@@ -158,11 +158,12 @@ func (s *SoftwareUpdater) Verify(version string, mirror string) error {
 		}
 	}
 
-	err = cmd.RunAndLog(args...)
-	if err != nil {
-		return errors.Wrap(err)
+	if !verifyOnly {
```

Comment:
Maybe move this up to ~136 and just return nil if verify only

Reply:
Done.